nautobot/ipam/graphql/types.py

```diff
@@ -12,7 +12,7 @@ class IPAddressType(gql_optimizer.OptimizedDjangoObjectType):
     family = graphene.Int()
     interface = graphene.Field("nautobot.dcim.graphql.types.InterfaceType")
     vminterface = graphene.Field("nautobot.virtualization.graphql.types.VMInterfaceType")
-    nat_outside = graphene.Field(lambda: IPAddressType)
+    nat_outside = graphene.List(lambda: IPAddressType)
```

Comment:
I _think_ this can be removed altogether.

It was added in https://github.com/nautobot/nautobot/pull/1420/files#diff-9687b347d7a3698e4a385d4fd99ceea3b22424c81f9ec9cdd74bc9a558ac30d1R43 because `nat_outside` was a method property so Graphene couldn't learn automagically. It should be able to look at the FKs and in fact this already works in 1.5: https://demo.nautobot.com/graphql/#query=%7B%0A%20%20ip_addresses(limit%3A%2010)%7B%0A%20%20%20%20nat_outside_list%20%7B%0A%20%20%20%20%20%20address%0A%20%20%20%20%7D%0A%20%20%7D%0A%7D

Reply:
Agreed - remove.